src/pep8_naming/checks.rs

```diff
@@ -74,6 +74,16 @@ pub fn invalid_first_argument_name_for_class_method(
         ),
         FunctionType::ClassMethod
     ) {
+        if let Some(arg) = args.posonlyargs.first() {
+            if arg.node.arg != "cls" {
+                return Some(Check::new(
+                    CheckKind::InvalidFirstArgumentNameForClassMethod,
+                    Range::from_located(arg),
+                ));
+            } else {
+                return None;
+            }
+        }
         if let Some(arg) = args.args.first() {
```

Comment:
Nit: can we make this an `else`, and remove the `return None` from line 84, to make the intent a bit clearer?